templates/zerver/help/delete-a-message.md

```diff
@@ -0,0 +1,7 @@
+# Delete a message
+
+{!admin-only.md!}
+{!message-actions.md!}
+{!down-chevron.md!}
+
+2. Select the **Delete message** option from the dropdown to delete that message.
```

Comment:
I think we should probably add a paragraph or two at the top of the form (maybe needs some editing):

"""
When you send a message that you didn't intend to send, there are a few options to fix it.  Often, the simplest solution is to just edit the message to correct the mistake.  If you intended to send nothing, deleting the content of the message in the editing UI will cause the message to be displayed as "(deleted)".  This often minimizes confusion if other users had responded to your original message, since they can see that you sent a message that was later deleted.  The original content of the message is still accessible via Zulip's edit history feature, so this is the appropriate approach in cases where the message was in error but did not contain secret information.

For cases where someone accidentally shared secret information publicly (E.g. you posted an employee's salary), Zulip does support permanently deleting a message, which is documented below.  Note, however, that anyone who received the message before you deleted it could have made a copy of its content.  So if you accidentally shared your password, you should change your password regardless of whether you delete the message.
"""